recipes/pygtftk/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "1.2.7" %}
+{% set version = "1.3.0" %}
 
 package:
   name: pygtftk
   version: '{{ version }}'
 
 source:
-  url: 'https://github.com/dputhier/pygtftk/archive/{{ version }}.tar.gz'
+  url: 'https://github.com/dputhier/pygtftk/archive/v{{ version }}.tar.gz'
   sha256: f3d58776a6ff3d79c43bf207a2157861ca2b5668f7f507dcca88994df6d77b4d
```

Comment:
Are you sure this hash was unchanged?

Reply:
You are right. I forgot to update the hash...